FhirToDataLake/src/Microsoft.Health.Fhir.Synapse.Common/Configurations/JobConfiguration.cs

```diff
@@ -30,16 +30,28 @@ public class JobConfiguration
         public string QueueUrl { get; set; }
 
         /// <summary>
-        /// Scheduler crontab expression.
+        /// Job information table name.
         /// </summary>
-        [JsonProperty("schedulerCronExpression")]
-        public string SchedulerCronExpression { get; set; }
+        [JsonProperty("jobInfoTableName")]
+        public string JobInfoTableName { get; set; }
```

Comment:
Shall we also update the deploy arm template to reflect this change?

Reply:
Updated, thanks.

Reply:
Maybe also `appsettings.json`?

Reply:
Updated